### ISSUE TITLE
OpenJS Foundation Cross Project Council Meeting 2020-08-11

### ISSUE THREAD

**Issue by mhdawson** [openjs-foundation/cross-project-council#615]
## Time

**UTC Tue 11-Aug-2020 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 11-Aug-2020 09:00 (09:00 AM) |
| US / Mountain | Tue 11-Aug-2020 10:00 (10:00 AM) |
| US / Central  | Tue 11-Aug-2020 11:00 (11:00 AM) |
| US / Eastern  | Tue 11-Aug-2020 12:00 (12:00 PM) |
| London        | Tue 11-Aug-2020 17:00 (05:00 PM) |
| Amsterdam     | Tue 11-Aug-2020 18:00 (06:00 PM) |
| Moscow        | Tue 11-Aug-2020 19:00 (07:00 PM) |
| Chennai       | Tue 11-Aug-2020 21:30 (09:30 PM) |
| Hangzhou      | Wed 12-Aug-2020 00:00 (12:00 AM) |
| Tokyo         | Wed 12-Aug-2020 01:00 (01:00 AM) |
| Sydney        | Wed 12-Aug-2020 02:00 (02:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Cross%20Project%20Council+Meeting+2020-08-11&iso=20200811T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+Aug+11%2C+2020+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1zXXqZcWJhivOvcr7KK879_tZMQzcKkeGOpENL21pbcU/edit>

## Agenda

Extracted from **cross-project-council-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/cross-project-council

* Next Steps for Supporter Program [#607](https://github.com/openjs-foundation/cross-project-council/issues/607)
* Director election process [#601](https://github.com/openjs-foundation/cross-project-council/issues/601)
* Develop annual review process for projects [#592](https://github.com/openjs-foundation/cross-project-council/issues/592)
* doc: Governance changes for Collaboration Network [#547](https://github.com/openjs-foundation/cross-project-council/pull/547)
* Add Foundation-wide copyright guidance [#414](https://github.com/openjs-foundation/cross-project-council/pull/414)
* Responsible security disclosures [#326](https://github.com/openjs-foundation/cross-project-council/issues/326)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `cross-project-council-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: https://zoom.us/j/920010234
* For those who just want to watch: https://livestream.openjsf.org

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by designMoreWeb** [comment]
I thought that we were going to have a different time for the cpc meeting as we wanted more engagement

**Comment by joesepi** [comment]
@designMoreWeb 

TL;DR
Next week we have an earlier time.

There has been some confusion around adding a third time and I think conversations during a recent meeting added to that. (I may have been part of that confusion generating team 😄  ). However, there is an issue where discussion took place and the time/date was determined: https://github.com/openjs-foundation/cross-project-council/issues/599
